src/adapters/bifrost.ts

```diff
@@ -193,6 +198,15 @@ class BaseBifrostAdapter extends BaseCrossChainAdapter {
       throw new CurrencyNotFound(token);
     }
 
+    // use "Unlimited" if the xToken.transfer's fourth parameter version supports it
+    const destWeight = supportsUnlimitedDestWeight(this.api)
```

Comment:
I don't think this makes much sense for bifrost since `DEST_WEIGHT = "Unlimited";` anyway - i.e. they must have already switched over to the new version, so we can remove the "future proofing" here

Reply:
Good catch, thank you! I didn't think of checking the weight in the config.